f32-py38/s2i_assemble.patch

```diff
@@ -31,11 +31,9 @@
 -# First of all, check that we don't have disallowed combination of ENVs
 -if [[ ! -z "$ENABLE_PIPENV" && ! -z "$ENABLE_MICROPIPENV" ]]; then
 -  echo "ERROR: Pipenv and micropipenv cannot be enabled at the same time!"
--  # podman/buildah does not relay this exit code but it will be fixed hopefuly
--  # https://github.com/containers/buildah/issues/2305
 -  exit 3
 -fi
```

Comment:
Haven't check this, will the path be still applicable?

Reply:
how can we check this? should I leave the comments for now?

Reply:
You can try to build the container image. This change might cause patch failure so worth to double check it.

Reply:
1. should we merge and try to build? or can we use /deploy here? I don't see .aicoe.yaml configured. 

2. Do we have docs on how to test builds of s2i-thoth images?